dev-requirements.txt

```
@@ -22,6 +22,6 @@ pytz
 tox>=3.2.0
 types-requests
 
-dbt-spark==1.4.*
-dbt-core==1.4.*
-dbt-tests-adapter==1.4.*
\ No newline at end of file
+dbt-spark==1.5.0b4
+# dbt-core==1.4.*
+dbt-tests-adapter==1.5.0rc1
```

can you try this? this basically installs based on the `setup.py` in dbt-core's `tests/adapter/` dir but from the `1.5.latest` branch.

my hunch is that something's goofy w/ `dbt-tests-adapter==1.5.0rc1` PyPI package...


```suggestion
git+https://github.com/dbt-labs/dbt-spark.git@1.5.latest#egg=dbt-spark
git+https://github.com/dbt-labs/dbt-core.git@1.5.latest#egg=dbt-core&subdirectory=core
git+https://github.com/dbt-labs/dbt-core.git@1.5.latest#egg=dbt-tests-adapter&subdirectory=tests/adapter
```